core/lowering/passes/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+target_sources(${lib_name} 
+    PRIVATE "${CMAKE_CURRENT_SOURCE_DIR}/convNd_to_convolution.cpp"
```

Comment:
Also here can we just glob the full directory *.cpp?

Reply:
see [my previous answer](https://github.com/pytorch/TensorRT/pull/1058#discussion_r872059785)